docs/paper/admin/reference/configuration/world-configuration.mdx

```diff
@@ -0,0 +1,740 @@
+
+          - **delete**: Delete the entity.
+
+          - **silent**: Does nothing, not printing logs.
+
+          - **warn**: Does nothing, printing logs'
+
+      safe-regen-delete-range:
+        default: '32'
+        description: 'If multiple entities with duplicate UUIDs are within this many
+          blocks, saferegen will delete all but 1 of them'
+    filter-bad-tile-entity-nbt-from-falling-blocks:
+      default: 'true'
+      description: 'Instructs the server to remove certain NBT data from falling blocks.
+        **Note**: Some adventure maps may require this to be turned off to function
+        correctly, but we do not recommend turning it off on a public server'
+    filtered-entity-tag-nbt-paths:
+      default: '[Pos, Motion, SleepingX, SleepingY, SleepingZ]'
+      description: 'A list of nbt tags that will be removed from the EntityTag tag
+        on items which spawn entities. The format of these strings follows the same
+        format used to select nbt tags in vanilla commands. If the spawning was directly
+        caused by a player and the player has the minecraft.nbt.place permission,
+        the filter list will be ignored. The defaults are set to prevent entities
+        from spawning or moving to a place other than the location they were placed.
+        For example, if Pos wasn''t included, a spawn egg could place an entity at
+        any location. **Note**: Some adventure maps may require this to be an empty
+        list to function correctly, but we do not recommend turning it off on a public
+        server'
+    iron-golems-can-spawn-in-air:
+      default: 'false'
+      description: 'Sets whether iron golems can spawn in the air. Iron farms may
+        break depending on this setting '
+    monster-spawn-max-light-level:
+      default: '-1'
+      description: 'When set to -1, the Vanilla default will be used (=0). Set to
+        15 or greater to revert to pre-1.18 behavior'
+    non-player-arrow-despawn-rate:
+      default: default
+      description: 'The rate, in ticks, at which arrows shot from non-player entities
+        are despawned. The default value instructs the server to use the same default
+        arrow despawn rate from spigot.yml that is used for all arrows'
+    per-player-mob-spawns:
+      default: 'true'
+      description: 'Determines whether the mob limit (in bukkit.yml) is counted per
+        player or for the entire server. Enabling this setting results in roughly
+        the same number of mobs, but with a more even distribution that prevents one
+        player from using the entire mob cap and provides a more single-player like
+        experience'
+    scan-for-legacy-ender-dragon:
+      default: 'true'
+      description: 'Determines if the server attempts to start the ender dragon fight.
+        Setting this to false will make the ender dragon not spawn in the end, even
+        with a new world'
+    skeleton-horse-thunder-spawn-chance:
+      default: '0.01'
```

Comment:
The default is now: default but maybe the real value could be written somewhere in the description? (could apply for others)